RELEASE-NOTES.md

```diff
@@ -1,3 +1,146 @@
+
+**1.0.0:** Subscription.request SHOULD respect the responsivity of its caller by returning in a timely manner[2].
+
+**1.0.1:** Subscription.request SHOULD respect the responsivity of its caller by returning in a timely manner.
+
+**Comment: Footnote reworked into the Intent-section of the rule.**
+
+## Subscription Rule 5
+
+**1.0.0:** Subscription.cancel MUST respect the responsivity of its caller by returning in a timely manner[2], MUST be idempotent and MUST be thread-safe.
+
+**1.0.1:** Subscription.cancel MUST respect the responsivity of its caller by returning in a timely manner, MUST be idempotent and MUST be thread-safe.
+
+**Comment: Footnote reworked into the Intent-section of the rule.**
+
+## Subscription Rule 13
+
+**1.0.0:** While the Subscription is not cancelled, Subscription.cancel() MUST request the Publisher to eventually drop any references to the corresponding subscriber. Re-subscribing with the same Subscriber object is discouraged [see 2.12], but this specification does not mandate that it is disallowed since that would mean having to store previously cancelled subscriptions indefinitely.
+
+**1.0.1:** While the Subscription is not cancelled, Subscription.cancel() MUST request the Publisher to eventually drop any references to the corresponding subscriber.
+
+**Comment: Second part of rule reworked into the Intent-section of the rule.**
+
+## Subscription Rule 15
+
+**1.0.0:** Calling Subscription.cancel MUST return normally. The only legal way to signal failure to a Subscriber is via the onError method.
+
+**1.0.1:** Calling Subscription.cancel MUST return normally.
+
+**Comment: Replaced second part of rule with a definition for `return normally` in the glossary.**
+
+## Subscription Rule 16
+
+**1.0.0:** Calling Subscription.request MUST return normally. The only legal way to signal failure to a Subscriber is via the onError method.
+
+**1.0.1:** Calling Subscription.request MUST return normally.
+
+**Comment: Replaced second part of rule with a definition for `return normally` in the glossary.**
+
+## Subscription Rule 17
+
+**1.0.0:** A Subscription MUST support an unbounded number of calls to request and MUST support a demand (sum requested - sum delivered) up to 2^63-1 (java.lang.Long.MAX_VALUE). A demand equal or greater than 2^63-1 (java.lang.Long.MAX_VALUE) MAY be considered by the Publisher as “effectively unbounded”[3].
+
+**1.0.1:** A Subscription MUST support an unbounded number of calls to request and MUST support a demand up to 2^63-1 (java.lang.Long.MAX_VALUE). A demand equal or greater than 2^63-1 (java.lang.Long.MAX_VALUE) MAY be considered by the Publisher as “effectively unbounded”.
+
+**Comment: Rule simplified by defining `demand` in the glossary, and footnote was reworked into the Intent-section of the rule.**
+
+---
+
+## TCK alterations
+
+- Fixed potential resource leaks in partially consuming Publisher tests
+- Fixed potential resource leaks in partially emitting Subscriber tests
```

Comment:
Link to https://github.com/reactive-streams/reactive-streams-jvm/pull/372  and https://github.com/reactive-streams/reactive-streams-jvm/pull/373